projects/schematics/src/migrations/3_0/constructor-deprecations/data/register.component.migration.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  REGISTER_COMPONENT,
+  SPARTACUS_STOREFRONTLIB,
+  ANONYMOUS_CONSENTS_SERVICE,
+  ANONYMOUS_CONSENTS_CONFIG,
+  AUTH_CONFIG_SERVICE,
+  FORM_BUILDER,
+  ANGULAR_FORMS,
+  ROUTING_SERVICE,
+  SPARTACUS_CORE
+} from '../../../../shared/constants';
+import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
+
+export const REGISTER_COMPONENT_MIGRATION: ConstructorDeprecation = {
+  // projects/storefrontlib/src/cms-components/user/register/review-submit.component.ts
+  class: REGISTER_COMPONENT,
+  importPath: SPARTACUS_STOREFRONTLIB,
+  deprecatedParams: [
+    {
```

Comment:
`UserService` and `GlobalMessageService` missing (were not visible on screenshot)